projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-speed-score.tsx

```diff
@@ -6,20 +6,58 @@ import {
 import { Spinner, BoostScoreBar } from '@automattic/jetpack-components';
 import { Popover } from '@wordpress/components';
 import { useViewportMatch } from '@wordpress/compose';
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 import { arrowUp, Icon } from '@wordpress/icons';
 import { useEffect, useState, useMemo, useCallback } from 'react';
 import { PRODUCT_STATUSES } from '../../../constants';
 import useProduct from '../../../data/products/use-product';
 import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
 import useAnalytics from '../../../hooks/use-analytics';
 import useMyJetpackConnection from '../../../hooks/use-my-jetpack-connection';
+import formatPercentage from '../../../utils/format-percentage';
 import { useBoostTooltipCopy } from './use-boost-tooltip-copy';
 import type { SpeedScores, BoostSpeedScoreType } from './types';
 import type { SetStateAction } from 'react';
 
 import './style.scss';
 
+const createSpeedScoreSRText = (
+	speedLetterGrade: string,
+	currentSpeedScore: number,
+	boostScoreIncrease?: number,
+	cta?: string
+) => {
+	const fragments = [];
+
+	fragments.push(
+		sprintf(
+			// translators: %1$s: speed grade (e.g. 'A'), %2$s: speed score percentage (e.g. '95%').
+			__( 'Your website’s overall speed score is %1$s, or %2$s.', 'jetpack-my-jetpack' ),
+			speedLetterGrade,
+			formatPercentage( currentSpeedScore / 100 )
+		)
+	);
+
+	if ( boostScoreIncrease ) {
+		fragments.push(
+			sprintf(
+				// translators: %s: score increase (e.g. '10')
+				__(
+					'Your website’s overall speed score is %s faster than the previous period.',
```

Comment:
@Automattic/jetpack-martech, do you know to what timeframe the difference in Boost score refers to? Last month? 30 days ago? If that information retrievable in the code?

Reply:
Page Speed scores are the current score of the website. I don't think there's anything to compare it to.

Do we need to mention that it's faster than a previous period? Can we instead just say that this is the overall speed score?

Reply:
The UI does mention an increase or decrease, so we need to include that information for screen readers, too. But whether this should be in the UI in the first place, I'll delegate this to @Automattic/jetpack-martech since they implemented the card in MyJP.